ibis/backends/bigquery/__init__.py

```diff
@@ -438,47 +438,82 @@ def create_table(
         database: str | None = None,
         temp: bool | None = None,
         overwrite: bool = False,
+        default_collate: str | None = None,
+        partition_by: str | None = None,
+        cluster_by: Iterable[str] | None = None,
+        options: Mapping[str, Any] | None = None,
     ) -> ir.Table:
         if obj is None and schema is None:
-            raise com.IbisError("The schema or obj parameter is required")
-        if temp is True:
-            raise NotImplementedError(
-                "BigQuery backend does not yet support temporary tables"
-            )
-        if overwrite is not False:
+            raise com.IbisError("One of the `schema` or `obj` parameter is required")
+
+        if temp:
+            # TODO: these require a BQ session; figure out how to handle that
             raise NotImplementedError(
-                "BigQuery backend does not yet support overwriting tables"
+                "Temporary tables in the BigQuery backend are not yet supported"
             )
 
+        create_stmt = "CREATE"
+
+        if overwrite:
+            create_stmt += " OR REPLACE"
+
+        table_ref = self._fully_qualified_name(name, database)
+
+        create_stmt += f" TABLE {table_ref}"
```

Comment:
Naive question - do we need backticks here? The other DDL calls all add them.

```suggestion
        create_stmt += f" TABLE `{table_ref}`"
```

Reply:
Yeah, we should add them here.

Reply:
Done!